ontology/rml-core.owl

```diff
@@ -0,0 +1,517 @@
+               rdfs:comment "Represents a Logical Target" ;
+               rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+               rdfs:label "Logical Target" .
+
+
+###  http://w3id.org/rml/ObjectMap
+:ObjectMap rdf:type owl:Class ;
+           rdfs:subClassOf :TermMap ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :datatype ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Datatype
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :object ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Resource
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :object ;
+                             owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                             owl:onClass rdfs:Resource
+                           ] ,
+                           [ rdf:type owl:Restriction ;
+                             owl:onProperty :language ;
+                             owl:minQualifiedCardinality "0"^^xsd:nonNegativeInteger ;
+                             owl:onDataRange xsd:string
+                           ] ;
+           rdfs:comment "Represents an object map."@en ;
+           rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+           rdfs:label "Object Map" .
+
+
+###  http://w3id.org/rml/PredicateMap
+:PredicateMap rdf:type owl:Class ;
+              rdfs:subClassOf :TermMap ,
+                              [ rdf:type owl:Restriction ;
+                                owl:onProperty :predicate ;
+                                owl:qualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                                owl:onClass rdf:Property
+                              ] ;
+              rdfs:comment "Represents a predicate map."@en ;
+              rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+              rdfs:label "Predicate Map" .
+
+
+###  http://w3id.org/rml/PredicateObjectMap
+:PredicateObjectMap rdf:type owl:Class ;
+                    rdfs:subClassOf owl:Thing ,
+                                    [ rdf:type owl:Restriction ;
+                                      owl:onProperty :predicateMap ;
+                                      owl:minQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                                      owl:onClass :PredicateMap
+                                    ] ,
+                                    [ rdf:type owl:Restriction ;
+                                      owl:onProperty :objectMap ;
+                                      owl:qualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                                      owl:onClass :ObjectMap
+                                    ] ;
```

Comment:
A `:PredicateObjectMap` can have one ore more `:PredicateMap`s and one or more `:ObjectMap`s. These should be removed.

Reply:
"These should be removed.": I don't see why, I'm ok with the minQualifiedCardinality for predicateMap, same should apply to objectMap, and object and predicate should be added here too.